### DISCUSSION TITLE
Grey line in the scatter plot

### DISCUSSION THREAD

**Discussion by deb0612** [etal/cnvkit#1137] [General]
Dear sir,
What's the meaning of grey plot in scatter plot.
 
![image](https://user-images.githubusercontent.com/29038270/103735491-26d05200-5029-11eb-8b06-f9d58745843e.png)
for example, in chr4 and chr8, the grey line shown here instead of orange line. 
How is that?

**Comment by etal** [comment]
If you plot a .call.cns, the gray lines are segments with neutral copy number (CN=2 in diploid samples, on autosomes).
